tests/e2e/pageObjects/types.ts

```diff
@@ -0,0 +1,18 @@
+export type StandardWalletType = '3-Key Vault' | '2-Key Vault' | 'Standard HD P2SH';
+
+export type WalletType = StandardWalletType | 'Standard P2SH' | 'Standard HD SegWit';
```

Comment:
(?) What do you think to change `StandardWalletType` to i.e. `BasicWalletType` or `DefaultWalletType` or something else? `Standard could bring some misunderstanding in names where you have 3 types of Standard wallets.

Reply:
Good point. Done